src/backer/backering.py

```diff
@@ -407,6 +411,8 @@ def __next__(self):
 
                     if self.ledger and topic == "/receipt":
                         try:
+                            self.queue,push_to_queued(self.pre, msg)
+
                             serder = serdering.SerderKERI(raw=msg)
                             event = eventing.loadEvent(self.hab.db, self.pre, serder.saidb)
                             self.ledger.publishEvent(event)
```

Comment:
We should no longer be pushing the event on `self.ledger` here if `queueing.py` does it

Reply:
updated